di_art/analysis_options.yaml

```diff
@@ -0,0 +1,14 @@
+# Defines a default set of lint rules enforced for
+# projects at Google. For details and rationale,
+# see https://github.com/dart-lang/pedantic#enabled-lints.
+include: package:pedantic/analysis_options.yaml
```

Comment:
It's not working if you won't specify the rules.
You can copy rules set from the logart project :)